File: Python/digital_sum.py
num=int(input("Enter the number"))
count=1
digital_sum=0
while(num>0 or digital_sum>=10):
    digital_sum=int(digital_sum+num%10)
    num=num/10
    if(digital_sum>=10 and num==0):
        num=digital_sum
        digital_sum=0
        count=count+1
print("Digital sum :",int (digital_sum))
print("Additive Parsistence :",count)

File: C/digitalsum.c
//find the digital sum of given number

#include<stdio.h>
int main(){
int digital_sum=0,num,count=1;
printf("Enter the number to find the digital sum ");
scanf("%d",&num); //enter the number 
while(num>0||digital_sum>=10){ 
	digital_sum=digital_sum+num%10;
	num=num/10;
	if(digital_sum>=10&&num==0){
		num=digital_sum;
		digital_sum=0;
		count++;
	}	
}
printf("Digital Sum : %d",digital_sum);
printf("\nAdditive Parsistence :%d",count);
}
